.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!-- Keep the progress checklist.  -->
 ## Progress
```

Comment:
I thought @mjhenkes was in favor of requiring all three- The progress checklist for quick reference, the project board we would maintain, and the labels for quick filtering and tracking purposes?